controllers/articles.controller.js

```diff
@@ -0,0 +1,12 @@
+const { selectArticleById } = require("../models/articles.model");
+
+exports.getArticleById = (req, res, next) => {
+  return selectArticleById(req)
+    .then(({ rows }) => {
+      if (!rows.length) {
+        return Promise.reject({ status: 404, msg: "Not found" });
+      }
+      res.status(200).send({ articles: rows });
```

Comment:
I would return this on a key of `article`, as you're only getting one article back, not articles 👍